netkan/netkan/notifications.py

```diff
@@ -23,7 +23,14 @@ def emit(self, record):
         self.webhook.send(f'```{fmt}```' if "\n" in fmt else fmt)
 
 
-def setup_log_handler():
+def setup_log_handler(debug=False):
+    if not sys.argv[0].endswith('gunicorn'):
```

Comment:
Gunicorn is one of several options for WSGI servers, our code should not care which one we are using.

Reply:
Considering we own and deploy the code on our infrastructure. As it's just for making console logging a little nicer, I don't think it matters too much.